Shared/Samples/Apply unique value renderer/ApplyUniqueValueRendererView.swift

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyUniqueValueRendererView: View {
+    /// A Boolean value indicating whether to show an alert.
+    @State private var isShowingAlert = false
+    
+    /// The error shown in the alert.
+    @State private var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    /// A map with topographic basemap centered on western United States.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        let centerPoint = Point(x: -12356253.6, y: 3842795.4, spatialReference: .webMercator)
+        map.initialViewpoint = Viewpoint(center: centerPoint, scale: 52681563.2)
+        
+        return map
+    }()
+    
+    /// Create a unique value renderer configured to render the Pacific states
+    /// as blue, the Mountain states as green, and the West South Central states
+    /// as brown.
+    private func makeUniqueValueRenderer() -> UniqueValueRenderer {
```

Comment:
Please redo the docstring with the template. If the suggestion above is applied, this becomes a `private static func`.

Reply:
Instead of setting the attributes separately, I'd like to first construct all the symbols, and then construct the renderer using it initializer.

```swift
    private static func makeUniqueValueRenderer() -> UniqueValueRenderer {
        let stateOutlineSymbol = SimpleLineSymbol(style: .solid, color: .white, width: 0.7)
        
        let pacificFillSymbol = SimpleFillSymbol(style: .solid, color: .blue, outline: stateOutlineSymbol)
        let mountainFillSymbol = SimpleFillSymbol(style: .solid, color: .green, outline: stateOutlineSymbol)
        let westSouthCentralFillSymbol = SimpleFillSymbol(style: .solid, color: .brown, outline: stateOutlineSymbol)
        
        let pacificValue = UniqueValue(description: "Pacific Region", label: "Pacific", symbol: pacificFillSymbol, values: ["Pacific"])
        let mountainValue = UniqueValue(description: "Rocky Mountain Region", label: "Mountain", symbol: mountainFillSymbol, values: ["Mountain"])
        let westSouthCentralValue = UniqueValue(description: "West South Central Region", label: "West South Central", symbol: westSouthCentralFillSymbol, values: ["West South Central"])
        
        let defaultFillSymbol = SimpleFillSymbol(style: .cross, color: .gray)
        
        return UniqueValueRenderer(
            fieldNames: ["SUB_REGION"],
            uniqueValues: [pacificValue, mountainValue, westSouthCentralValue],
            defaultLabel: "Other",
            defaultSymbol: defaultFillSymbol
        )
    }
```